Justfile

```diff
@@ -0,0 +1,28 @@
+set shell := ["bash", "-euo", "pipefail", "-c"]
+
+export DOCKER_CLI_HINTS := "false"
+
+image_name := "ghcr.io/githedgehog/hhdocs"
+image_version := "20241202"
+image := image_name + ":" + image_version
+
+_run +cmd:
+  docker run -q --pull=always --rm -v $(pwd):/docs -p 8000:8000 {{image}} {{cmd}}
```

Comment:
Unless I'm mistaken, this will be the default command if the user types `just`, and this is probably not what we want.

How about adding one of these above:

```justfile
[private]
@default: build
```

or

```justfile
# List out the available commands
[private]
@default:
    just --list --justfile {{ justfile() }}
```